src/applications/personalization/dashboard-2/tests/unit/healthcare.unit.spec.jsx

```diff
@@ -9,6 +9,52 @@ import HealthCare from '~/applications/personalization/dashboard-2/components/he
 describe('HealthCare component', () => {
   let view;
   let initialState;
+
+  context('when appointments and messaging data are still loading', () => {
+    it('should only show a loading spinner', async () => {
+      window.VetsGov = { pollTimeout: 1 };
+      mockFetch();
+      initialState = {
+        user: {
+          profile: {
+            services: ['messaging'],
+          },
+        },
+        health: {
+          appointments: {
+            fetching: true,
+          },
+          msg: {
+            folders: {
+              data: {
+                currentItem: {
+                  loading: true,
+                },
+              },
+              ui: {
+                nav: {
+                  foldersExpanded: false,
+                  visible: false,
+                },
+              },
+            },
+          },
+        },
+      };
+
+      view = renderInReduxProvider(<HealthCare dataLoadingDisabled />, {
+        initialState,
+        reducers,
+      });
+      expect(view.queryByRole('progressbar')).to.exist;
+      expect(view.queryByText('Refill and track your prescriptions')).not.to
```

Comment:
I've learned to always use a "looser" regex when making sure that elements are _not_ there. Doing a strict text match like this is much more likely to not find an element even if what it's looking for exists. For example, if the DOM contained an element with the exact contents of: `Refill and track your prescriptions.` (note the trailing period), this query would still succeed (_not_ find the element). tldr, this should be searching for `/Refill and track your prescriptions/i`

Reply:
Okay, that makes sense! I made the change :)